### PR TITLE
Navigate the search results list and access the profile

### PR DESCRIPTION
**Pull request by nabil-dbz**
- The user is able to navigate through the list of elements using the mouse or the keyboard
- The element highlighted, background lightgray, is the one to be access when the enter key is pressed
- Add the user's profile picture in the element
- The user can use the key arrows to go through the list of elements
- The maximum number of elements in the result list is 5 (can be changed)

### CODE REVIEW

File: planet/src/main/webapp/js/search-bar.js [hunk 1]
```diff
@@ -12,29 +12,76 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+const UP_ARROW = 38;
+const DOWN_ARROW = 40;
+const ENTER_KEY = 13;
```
Comment (azhang72) [inline planet/src/main/webapp/js/search-bar.js]: Where are these key codes defined?

Comment (nabil-dbz) [reply planet/src/main/webapp/js/search-bar.js]: In the search-bar script as global constants. Should I put them inside the function or in a separate file?

Comment (azhang72) [reply planet/src/main/webapp/js/search-bar.js]: I assume you got these key codes from some standard and didn't come up with the values yourself? Where is that standard and can you directly reference the key values from there? That way, your keyboard navigation won't break if someday the key values change.

Comment (nabil-dbz) [reply planet/src/main/webapp/js/search-bar.js]: I actually got the keycodes form this page: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode
I just realized that it's inadvisable so I'll go for another simpler way: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key

Comment (nabil-dbz) [reply planet/src/main/webapp/js/search-bar.js]: Done!

**Review by EkkiD — APPROVED** [review]
LGTM

**Review by azhang72 — APPROVED** [review]

**Review by emilyzhaoo — APPROVED** [review]
LGTM

**Pull request merged**